tendrl/monitoring_integration/grafana/alert_dashboard.py

```diff
@@ -26,7 +26,7 @@ def set_alert(panel, thresholds, severity, resource_name, title):
              "type": "query"
              }],
             "executionErrorState": "keep_state",
-            "frequency": "60s", "handler": 1,
+            "frequency": "75s", "handler": 1,
```

Comment:
what I observed from testing is, it is always better to keep different frequency value critical and warning alert panels.

Reply:
Any regressions due to this and what is side effect of this?

Reply:
i tested this thoroughly it works fine